packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/index.tsx

```diff
@@ -1,30 +1,38 @@
-import { Popover2 } from '@blueprintjs/popover2';
-import { Button } from '@mantine/core';
+import { Button, Popover } from '@mantine/core';
 import { IconChevronDown } from '@tabler/icons-react';
-import React from 'react';
 import useEcharts from '../../../hooks/echarts/useEcharts';
-import { COLLAPSABLE_CARD_BUTTON_PROPS } from '../../common/CollapsableCard';
+import {
+    COLLAPSABLE_CARD_BUTTON_PROPS,
+    COLLAPSABLE_CARD_POPOVER_PROPS,
+} from '../../common/CollapsableCard';
 import MantineIcon from '../../common/MantineIcon';
 import ChartConfigTabs from './ChartConfigTabs';
 
 const ChartConfigPanel: React.FC = () => {
     const eChartsOptions = useEcharts();
     const disabled = !eChartsOptions;
-
     return (
-        <Popover2
+        <Popover
+            withinPortal={true}
```

Comment:
Yea, this was a late addition :). It's because when you click 'Flip axes' on the bar chart, it becomes a horizontal bar chart; that causes the buttons to change size, and the popup jumps around a bit. Putting it in the portal fixes this. The other configs don't need this cause they don't have buttons that change the page layout while the panel is open. 

FWIW, this is kind of what the portal is for -- to take an element out of page layout and make it stay put. We have this panel as a dropdown at the moment, but it's really more like a persistent modal.